ext/curl/interface.c

```diff
@@ -1155,6 +1219,17 @@ PHP_FUNCTION(curl_init)
 }
 /* }}} */
 
+static void _php_copy_callback(php_curl *ch, php_curl_callback **new_callback, php_curl_callback *source_callback, CURLoption option)
```

Comment:
`static zend_always_inline`? (same on the "free" one)

Reply:
I'm not sure. I think not forcing inlining is better because it's not a hot code path, so it doesn't make sense sacrificing instruction cache space by inlining this. If the compiler deems it performance critical to inline this, it will likely do that.